code/modules/mob/living/carbon/human/life.dm

```diff
@@ -850,6 +851,49 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 
 	return //TODO: DEFERRED
 
+/mob/living/carbon/human/proc/handle_addictions()

```

Comment:
There seems to be a lot of copypasta going on between all the Life() files and in this handle_addictions() proc. I think it's better to move these procs to a /mob/living level, and then determine what should(n't) run based on mob variables.